bindings/python/MANIFEST.in

```diff
@@ -1,14 +1,21 @@
 include README.rst
 include LICENSE
 include build-libbson.sh
-include release.sh
+include pyproject.toml
+
+exclude THIRD-PARTY-NOTICES
+exclude release.sh
+exclude addtags.py
+exclude benchmark.py
 
 graft pymongoarrow
 
-prune test
+recursive-exclude test *
```

Comment:
It appears from the logs that test test/ dir now gets included in the wheel build. Is somethign off here?
```
copying build/lib.linux-x86_64-3.10/pymongoarrow/bson.pyi -> build/bdist.linux-x86_64/wheel/pymongoarrow
creating build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_schema.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_arrow.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_pymongoarrow.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/__init__.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_datetime.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_monkey.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/utils.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_builders.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_pandas.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_numpy.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_libbson.py -> build/bdist.linux-x86_64/wheel/test
copying build/lib.linux-x86_64-3.10/test/test_bson.py -> build/bdist.linux-x86_64/wheel/test
...
```
https://github.com/mongodb-labs/mongo-arrow/runs/4527432236?check_suite_focus=true#step:3:607

Reply:
Fixed in latest push